### PR TITLE
Update to rules format 1.1

### PR DESCRIPTION
**Pull request by WilcoFiers**
- Move assumptions & accessibility support section to the background
- Add a `rules_format` property to the page front matter
- Update the outcome definition
- Update the tests
- Update the templates

Need for Call for Review: 2 weeks, this is a substantial change

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by carlosapaduarte — APPROVED** [review]
The preamble of the rule-design document (pages/design/rule-design.md) still had a mention of the rule format 1.0. I've updated it with [this commit](https://github.com/act-rules/act-rules.github.io/pull/2220/commits/ca3955abcc836d3ad7fafac4bbe201d17888aad2)

**Review by dan-tripp-siteimprove — APPROVED** [review]
I found some references to rules format 1.0 in some examples, in the files _rules/html-page-lang-matches-default-ucwvc8.md , _rules/image-no-text-0va7u6.md , and pages/examples/element-language.md.  Not sure if you care about that.

File: pages/design/rule-design.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
 title: Rule Design
 ---
 
-The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is based on [ACT Rules Format 1.0](https://www.w3.org/TR/act-rules-format/):
+The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is based on [ACT Rules Format 1.1](https://www.w3.org/TR/act-rules-format/):
```
Comment (WilcoFiers) [inline pages/design/rule-design.md]:
```suggestion
The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is based on [ACT Rules Format](https://www.w3.org/TR/act-rules-format/):
```

**Review by HelenBurge — COMMENTED** [review]
Looks good to me!

**Review by HelenBurge — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

File: pages/design/rule-template.md [hunk 2]
```diff
@@ -74,6 +67,26 @@ There are no accessibility support issues known.
...
+
+There are no accessibility support issues known.
+
+### Related Rules
+
+<!-- This section is optional -->
+
+- [rule name here](./abc123)
+
+### Bibliography
```
Comment (kengdoj) [inline pages/design/rule-template.md:84]: I think "Bibliography" under Background should be ["Other Resources"](https://w3c.github.io/wcag-act/act-rules-format.html#other-resources).

File: pages/glossary/outcome.md [hunk 3]
```diff
@@ -5,12 +5,21 @@ unambiguous: true
 objective: true
 ---
 
-An _outcome_ is a conclusion that comes from evaluating an ACT Rule on a [test subject](https://www.w3.org/TR/act-rules-format/#test-subject) or one of its constituent [test target](https://www.w3.org/TR/act-rules-format/#test-target). An outcome can be one of the three following types:
+A conclusion that comes from evaluating an ACT Rule on a [test subject][] or one of its constituent test target. An outcome can be one of the five following types:
```
Comment (kengdoj) [inline pages/glossary/outcome.md:8]: The format of other [glossary](https://act-rules.github.io/glossary/) definitions include the _term is_ so suggesting that stay in:

```suggestion
An _outcome_ is a conclusion that comes from evaluating an ACT Rule on a [test subject][] or one of its constituent test target. An outcome can be one of the five following types:
```

File: pages/glossary/outcome.md [hunk 4]
```diff
@@ -5,12 +5,21 @@ unambiguous: true
...
 
 - **Inapplicable:** No part of the test subject matches the applicability
-- **Passed:** A [test target](https://www.w3.org/TR/act-rules-format/#test-target) meets all expectations
-- **Failed:** A [test target](https://www.w3.org/TR/act-rules-format/#test-target) does not meet all expectations
+- **Passed:** A [test target][] meets all expectations
+- **Failed:** A [test target][] does not meet all expectations
+- **cantTell:** Whether the rule is applicable, or not all expectations were met could not be fully determined by the tester.
```
Comment (kengdoj) [inline pages/glossary/outcome.md:13]:
```suggestion
- **cantTell:** Whether the rule is applicable, or whether all expectations were met could not be fully determined by the tester.
```

File: pages/glossary/outcome.md [hunk 5]
```diff
@@ -5,12 +5,21 @@ unambiguous: true
...
+- **Untested**: The tester has not attempted to evaluate the test subject.
```
Comment (kengdoj) [inline pages/glossary/outcome.md:14]:
```suggestion
- **Untested**: The test subject was not evaluated for the rule.
```

**Review by kengdoj — CHANGES_REQUESTED** [review]

File: pages/glossary/outcome.md (hunk 3 above)
Comment (WilcoFiers) [reply pages/glossary/outcome.md:8]: This is taken from the new outcome definition: https://www.w3.org/TR/act-rules-format-1.1/#outcome

Comment (WilcoFiers) [reply pages/glossary/outcome.md:8]: Discussed in TF.

File: pages/glossary/outcome.md (hunk 4 above)
Comment (WilcoFiers) [reply pages/glossary/outcome.md:13]: We should make this change in the rules format first.

**Review by kengdoj — APPROVED** [review]

**Pull request merged**